common/context/context.cpp

```diff
@@ -14,16 +14,28 @@
 
 #include <common/context.hpp>
 
-#include <common/error.hpp>
+#include <common/common.hpp>
 #include <common/conf/paths.hpp>
+#include <common/error.hpp>
+#include <common/expected.hpp>
+#include <common/io.hpp>
+#include <common/json.hpp>
+#include <common/key_value_database.hpp>
 
 namespace mender {
 namespace common {
 namespace context {
 
 using namespace std;
+namespace common = mender::common;
 namespace conf = mender::common::conf;
 namespace error = mender::common::error;
+namespace expected = mender::common::expected;
+namespace io = mender::common::io;
+namespace json = mender::common::json;
+namespace kv_db = mender::common::key_value_database;
+
+const string MenderContext::provides_key_ = "artifact-provides";
```

Comment:
@kacf no idea what the right key name here is. I guess it should be consistent with the Go client?